package.json

```diff
@@ -9,6 +9,9 @@
     "run-tests": "lab -t 85 -e 'development'",
```

Comment:
Instead of ignoring the tests ... lets just ignore the leaks here with the flag example `--ignore SharedArrayBuffer,Atomics`

Reply:
👍